.changeset/fix-docs-scroll.md

```diff
@@ -0,0 +1,5 @@
+---
+'react-magma-docs': patch
+---
+
+fix(docs): Fix auto-scrolling to the selected component in the documentation on the first render/on a new tab.
```

Comment:
Please update the PR title to match this so we can keep track of what this is :)